addons/project/wizard/project_template_create_wizard.xml

```diff
@@ -17,8 +17,15 @@
                         <field name="date" required="not template_has_dates and date_start" invisible="1"/>
                         <field name="date_start" required="not template_has_dates and date" string="Planned Date" widget="daterange" options='{"end_date_field": "date", "always_range": "1"}'/>
                         <label for="alias_name" string="Create tasks by sending an email to" class="pe-2"/>
-                        <div class="d-inline-flex">
-                            <field name="alias_name" placeholder="e.g. office-party"/>@ <field name="alias_domain_id" placeholder="e.g. mycompany.com" options="{'no_create': True, 'no_open': True}"/>
+                         <div class="row" name="alias_email">
```

Comment:
This one suffers from the same issue: it's the last field of the form, meaning the copy will likely not be shown.

In addition, it requires adding a field and copying logic from mail alias mixin.

I would add this as an argument to not do it in either (not in project.project form view and not in this wizard) to have a consistent behavior and lessen code footprint. It just doesn't seem worth the trouble.